apps/app-article-server/src/browser/components/article-content.js

```diff
@@ -20,9 +20,7 @@ class Content extends Component {
       if (blockType === "ad") {
         return this.renderAd(block, key);
       }
-      if (blockType === "paragraph") {
-        return this.renderParagraph(block, key);
-      } else if (blockType === "headline") {
```

Comment:
One of the very few non-formatter changes: `htmlToReactParser` is always `undefined` in the `renderParagraph` function, so calling `renderParagraph` would always fail -- since it hasn't been a problem, we aren't rendering any blocks with the type being `paragraph`.